test/tla/cli-integration-tests.md

```diff
@@ -1998,7 +2002,9 @@ EXITCODE: OK
 ### set out-dir by CLI flag
 
 ```sh
-$ apalache-mc check --out-dir=./test-out-dir --length=0 Counter.tla &> /dev/null && find ./test-out-dir -type f -exec basename {} \; | ./sort.sh
+$ apalache-mc check --out-dir=./test-out-dir --length=0 Counter.tla | sed 's/[IEW]@.*//' && find ./test-out-dir -type f -exec basename {} \; | ./sort.sh
+...
+EXITCODE: OK
```

Comment:
These changes just let us get more helpful feedback when the outputs fail due to a crash or something.